src/main/resources-stable/META-INF/experiments.xml

```diff
@@ -27,5 +27,8 @@
         <experimentalFeature id="org.rust.crates.local.index" percentOfUsers="0">
             <description>Enables crates local index</description>
         </experimentalFeature>
+        <experimentalFeature id="org.rust.wsl.toolchain.flavor" percentOfUsers="100">
```

Comment:
Let's disable it for now in stable because we have some issues with it but I want to merge these changes.
I think we should ask @lancelote to check the feature after merge and if he doesn't find any critical issues, turn it on